org.civicrm.reqangular/src/common/modules/compuRoute.js

```diff
@@ -0,0 +1,7 @@
+define([
+    'common/angular'
+], function (angular) {
+    'use strict';
+
+    return angular.module('common.compuRoute', []);
```

Comment:
Unless you see the possibility of future components all linked together by a "compuRoute" context (like we have with `common.apis` and `common.dialog`), I think we can simply use `common.services` as the module for the decorator